accounts/views.py

```diff
@@ -177,24 +202,6 @@ def get_context_data(self, **kwargs):
         return context
 
 
-class UserProfileUpdate(LoginRequiredMixin, UpdateView):
```

Comment:
@autn interesting - was this already available? Is there a reason not to use this class-based `UpdateView` and instead use the function-based view? I'm OK with either, just wondering.

Reply:
@YPCrumble this class was copied from Omnifollow, and we need to adjust it to make it work correctly (it is unused on Omnifollow). I used the function-based view because I want the view to be the same as the signup page.